scripts/eurostat/gdp/import_data.py

```diff
@@ -82,8 +87,23 @@ def clean_data(self):
                              "before clean_data.")
         self.clean_df = self.preprocessed_df[self.DESIRED_COLUMNS]
 
-        # Prepends nuts/ prefix to geo codes.
-        self.clean_df['geo'] = self.clean_df['geo'].apply(lambda g: "nuts/" + g)
+        # GDP measurements for all of Europe are currently removed for lack
+        # of a way to represent them in the DataCommons Graph.
+        # TODO(fpernice): Add Europe-wide data to the import once it's
+        # supported by DataCommons.
+        self.clean_df = self.clean_df[~self.clean_df['geo'].isin(['EU27_2020',
+                                                                  'EU28'])]
+
+        def geo_converter(geo):
+            """Converts geo codes to nuts or country codes."""
+            if any(char.isdigit() for char in geo):
+                return 'nuts/' + geo
+            return COUNTRY_MAP.get(geo, "")
+
+        # Convert geo IDS to geo codes, e.g., "country/SHN" or "nuts/AT342".
+        self.clean_df['geo'] = self.clean_df['geo'].apply(geo_converter)
+        # Remove geos that do not adjust to any of the recognized standards.
+        self.clean_df = self.clean_df[self.clean_df['geo'] != ""]
```

Comment:
For record keeping, can you provide a count of how many this removes, or if not many, a list? It's possible that we need to add to the COUNTRY_MAP.

Reply:
Added a print statement that reports this information. Output of latest run was 
```
Num invalid geos: 1729 out of 38456 total geos.
```